File: src/Person.java
import javax.xml.bind.annotation.*;

@XmlRootElement(name = "Person")
@XmlAccessorType(XmlAccessType.FIELD)
public class Person {
    private String name;
    private String birthday;

    public Person() {
    }

    public Person(String name, String birthday) {
        this.name = name;
        this.birthday = birthday;
    }

    public String getPersonName() {
        return name;
    }

    public void setPersonName(String name) {
        this.name = name;
    }

    public String getPersonBirthDay() {
        return birthday;
    }

    public void setEmpName(String birthday) {
        this.birthday = birthday;
    }

}

File: src/main/kotlin/ru/cib/Person.kt
package ru.cib

import java.util.*
import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlElementWrapper
import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter


@XmlAccessorType(XmlAccessType.FIELD)
class Person {
    @XmlElement var name: String? = null
    @XmlJavaTypeAdapter(DateAdapter::class)
    var birthday: Date? = null
    @XmlElementWrapper(name = "hobbies")
    @XmlElement(name = "hobby")
    var hobbies: List<Hobby>? = null

    override fun toString(): String {
        return "Name: $name, Birthday: $birthday, Hobby: $hobbies"
    }
}